sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/google/GoogleHttpClient.java

```diff
@@ -61,6 +62,8 @@
     private static final String ERROR_CODE_GLOBAL_TOKEN_EXPIRED = "VALIDATION_TOKEN_EXPIRED_GLOBAL_TIME_OUT";
 
     private static final int HTTP_REQUEST_RETRY_COUNT = 1;
+    private static final int HTTP_REQUEST_DEFAULT_READ_TIMEOUT = 60000;
```

Comment:
Perhaps this should be read from the OctaneCustomSettings class.  There should be one single source of truth